README.md

```diff
@@ -111,23 +111,43 @@ Next / interim revision example:
 
 The generated `.vsix` will be under folder `./dist`
 
-## Publishing vscode-builtin-extensions to open-vsx
+## Packaging builtin-extensions as a builtin-extension-pack '.vsix`
 
-The `ovsx` client is used to publish to an open-vsx registry. By default https://open-vsx.org is used. To use another open-vsx registry, set its URL in environment variable `OVSX_REGISTRY_URL`. Use enviromnent variable `OVSX_PAT` to store the personal access token obtained from the registry.
+The version of the builtin extension-pack corresponds to the `version` present in the vscode submodule's `package.json`. For `next` versions, an appropriate hash suffix is added.
 
-After packaging the extensions as `.vsix` (see above), you may examine/test them under folder `dist`. Remove any that you do not wish to be published (e.g. those not working well). When ready proceed with publishing:
+Latest / solid revision example:
+
+    cd vscode; git checkout 1.45.0; cd ..
+    yarn; yarn create-extension-pack:latest
+
+Next / interim revision example:
+
+    cd vscode; git checkout d69a79b73808559a91206d73d7717ff5f798f23c; cd ..
+    yarn; yarn create-extension-pack:next
+
+The generated `.vsix` will be under the folder `./dist`
+
+## Publishing vscode-builtin-extensions and builtin-extension-packs to open-vsx
```

Comment:
I suggest "Publishing the built-in vscode extension pack to open-vsx"

Reply:
The thing is that this step publishes both, extension and extension-packs, so it seems the logical title would be: 
`Publishing the built-in vscode extensions and builtin-extension-packs to open-vsx`

Reply:
Actually, combining the comment with the next:
`Publishing individual built-in vscode extensions and builtin-extension-packs to open-vsx`
note that I am proposing to keep the plural form as we most likely be publishing more than one package in the near future.